Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -192,54 +202,35 @@ private function mapIncludeFilter(Store $store, IncludeFilters &$includeFilters,
                 $value = (bool)$item->getData('value');
                 break;
             default:
-                $this->logger->debugWithSource(
-                    sprintf(
-                        'Cannot build include filter for "%s" frontend input type',
-                        $frontendInput
-                    ),
-                    [],
-                    $this
-                );
-                return;
-        }
-        try {
-            $attributeCode = $attributeModel->getAttributeCode();
-            if (!is_string($attributeCode)) {
-                $this->logger->debugWithSource(
-                    sprintf(
-                        'Cannot build include filter for "%s" attribute ',
-                        $attributeModel->getName()
-                    ),
-                    [],
-                    $this
-                );
-                return;
-            }
-            $this->mapValueToFilter($includeFilters, $store, $attributeCode, $value);
-        } catch (NostoException $e) {
-            $this->logger->exception($e);
+                throw new NotSupportedFrontedInputException($store, $frontendInput);
```

Comment:
Shouldn't the previous exception be passed into the `NotSupportedFrontedInputException` so we can get the exception hierarchy? https://www.php.net/manual/en/exception.getprevious.php

Reply:
It's not being thrown when catching an exception. It might be a little confusing looking at the changes